ostree/ostree_dest.go

```diff
@@ -11,12 +11,15 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/containers/image/manifest"
 	"github.com/containers/image/types"
 	"github.com/containers/storage/pkg/archive"
 	"github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
+
+	"github.com/ostreedev/ostree-go/pkg/otbuiltin"
```

Comment:
so this is now importing cgo code if I've understood correctly, and as @mtrmac pointed out, we also need a way to opt in to have this (as opposed as to have this cgo by default). Would it make sense to have a `ostree_dest_nocgo.go` which still uses the non-c bindings? Or well, just have build tags at least?